runtimes/go-1.22/helpers/after-build.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+# Fail build if any command fails
+set -e
+
+echo "Compiling ..."
+
+# Compile the Code
+cd /usr/local/server/src
+go get openruntimes/handler@v0.0.0
+go build
```

Comment:
```suggestion
go build -ldflags="-s -w"
```
this will build for production stripping off any off information not needed during execution